var/spack/repos/builtin/packages/py-rioxarray/package.py

```diff
@@ -16,13 +16,25 @@ class PyRioxarray(PythonPackage):
 
     license("Apache-2.0")
 
+    version("0.17.0", sha256="46c29938827fff268d497f7ae277077066fcfbac4e53132ed3d4e2b96455be62")
     version(
         "0.4.1.post0", sha256="f043f846724a58518f87dd3fa84acbe39e15a1fac7e64244be3d5dacac7fe62b"
     )
 
-    depends_on("python@3.7:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-rasterio", type=("build", "run"))
+
+    # for optional interp, let numpy control scipy version
     depends_on("py-scipy", type=("build", "run"))
```

Comment:
If this is optional we should either add a variant or remove it

Reply:
Ok. I have done so, and set it to true to continue the existing behaviour of this package.